cmd/tether/ops_linux.go

```diff
@@ -126,9 +133,47 @@ func (t *operations) Setup(sink tether.Config) error {
 		}
 	}
 
+	// NOTE: ulimit default values should change when we support ulimit configuration
+	setupDefaultULimit()
+
 	return nil
 }
 
+// set ulimit fields to unlimited as their default value
+func setupDefaultULimit() {
+	var rLimit syscall.Rlimit
+
+	// NOFILE does not support defaultULimit as a value due to kernel restriction on number of open files
+	rLimit.Max = defaultNOFILE
+	rLimit.Cur = rLimit.Max
+	err := syscall.Setrlimit(syscall.RLIMIT_NOFILE, &rLimit)
```

Comment:
just cosmetic but all those if statements can be collapsed into single line form (if err:= ...; err != nil {...})